packages/react-native-bpk-component-rating/src/ios/RatingBridge/RCTBPKRating.h

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
I don't think we're meant to use the `RCT` namespace 🤔

Reply:
It's `RCTBPK` not `RCT` we use that for all extensions of Backpack with the purpose of supporting RN

Reply:
👍